source/_patterns/00-atoms/block/heading/heading.twig

```diff
@@ -1,5 +1,5 @@
 <h{{ level|default(1) }} {%- include 'atoms-attributes' %}>
 
-  {%- include 'atoms-text' with {nodes: text} only -%}
+  {%- include 'atoms-link' -%}
```

Comment:
What is the appropriate way to set the attributes for the `link` template?

Reply:
Because link is supported as an inline element it would be possible to introduce the link in the data as:

```
title:
  text:
    - template: atoms-link
      arguments:
        attributes:
          href: '#'
        text: Teaser title
```

Reply:
> What is the appropriate way to set the attributes for the `link` template?

If we leave like this then the `link` template will pick up the attributes intended for the `heading` template.

Reply:
Options that spring to mind:

- Have `href` as a top-level piece of data that's then used by items that link (title and image, though technically you could make more clickable).
- Do something like https://github.com/libero/pattern-library/blob/a292fe6714fdcfacf118cc5bd777f57e2307b5d5/source/_patterns/01-molecules/tag-list/tag-list.twig#L41-L45 in the `heading` pattern (depends if we always want links inside headings to look like the text)
- Don't use the `heading` pattern in this case (since it's simple HTML, it's more of a helper than a requirement)